scripts/create-darwin-volume.sh

```diff
@@ -89,9 +73,7 @@ test_t2_chip_present(){
 }
 
 test_filevault_in_use() {
-    disk=$1
-    #      list vols on disk | get value of Filevault key | value is true
-    apfs_volumes_for "$disk" | volume_list_true FileVault | grep -q true
+    fdesetup isactive
```

Comment:
This command doesn't exist on macOS catalina. Could we keep `volume_list_true` to continue supporting it?

Reply:
We'll obviously keep whatever works, but can you double-check? I have it on Catalina (and Mojave, and Big Sur) at `/usr/bin/fdesetup`:
```
$ sw_vers -productVersion && fdesetup isactive
10.15.5
false
```

Reply:
I also triggered a travis-ci build to check availability for this on a span of available versions from 10.12.6 through 10.15 at https://travis-ci.com/github/nix-community/nix-travis-ci/builds/183399921. These are "failing" because fdesetup isactive returns 1, but they all appear to have the command so far.

Reply:
Ahh sorry! Yeah I have it. I had been trying fdsetup :)

Reply:
No worries. Happy to have you giving it a shake :)

Reply:
Does this need testing for T1/no T1 conditions or did you already do that?